doc/system_requirements.md

```diff
@@ -0,0 +1,111 @@
+# System Requirement Specification Exasol Virtual Schema Common for Lua
+
+## Introduction
+
+Exasol Virtual Schema Common for Lua (short "EVSCL") is base library for [Virtual Schemas](https://docs.exasol.com/db/latest/database_concepts/virtual_schemas.htm) that connect to Exasol databases. 
+
+Virtual Schemas are conceptually related to database views. The main difference compared to database views is that Virtual Schema sources can be almost any data source. To make this possible, you need a so-called "Virtual Schema adapter", an Exasol extension that contains the logic to translate between the source and the projection provided by the Virtual Schema.
+
+From an end-user's perspective a Virtual Schema looks and feels like an internal schema of Exasol &mdash; albeit one that can only be read.
+
+## About This Document
+
+### Target Audience
+
+The target audience are end-users, requirement engineers, software designers and quality assurance. See section ["Stakeholders"](#stakeholders) for more details.
```

Comment:
requirement engineers, software designers and quality assurance are not listed in the "Stakeholder" section

Reply:
Added.